File: obg.Domain/Entities/Session.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace obg.Domain.Entities
{
    public class Session
    {
        public string IdSession { get; set; }
        public string UserName { get; set; }
        public string Token { get; set; }

        public Session(string idSession, string userName, string token)
        {
            IdSession = idSession;
            UserName = userName;
            Token = token;
        }
    }
}

File: ObgApp/src/app/models/session.ts
export class Session {
    idSession: string | null;
    userName: string | null;
    token: string | null;

    constructor(idSession: string | null, userName: string | null, token: string | null) {
        this.idSession = idSession;
        this.userName = userName;
        this.token = token;
    }
}